doc/help/MV.md

```diff
@@ -0,0 +1,23 @@
+kdb-mv(1) -- Move keys within the key database
+==============================================
+
+## DESCRIPTION
+
+This command moves key(s) in the Key database.  
+You can move keys to another directory within the database or even below another key.  
+
+## USAGE
+
+`kdb mv <source> <dest>`  
+
+Where `source` is the path of the key(s) you want to copy and `dest` is the path where you would like to move the key(s) to.  
+
+## EXAMPLES
+
+To move a KeySet:  
```

Comment:
@markus2330 Do you have a preferred terminology for these kinds of cases? I know we used to use KeySet but I understand how it can be misleading so do you have suggestions on what we should call multiple keys under a key?